commit 325f5f0ec83190f5da79a46600247e8dafc2fae6
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Jul 14 22:02:09 2014 +0100

    hid: Reduce default mouse polling interval to 60Hz
    
    Reduces overhead when using X

diff --git a/drivers/hid/usbhid/hid-core.c b/drivers/hid/usbhid/hid-core.c
index 77c50cdfff97..84759c49c880 100644
--- a/drivers/hid/usbhid/hid-core.c
+++ b/drivers/hid/usbhid/hid-core.c
@@ -48,7 +48,7 @@
  * Module parameters.
  */
 
-static unsigned int hid_mousepoll_interval;
+static unsigned int hid_mousepoll_interval = ~0;
 module_param_named(mousepoll, hid_mousepoll_interval, uint, 0644);
 MODULE_PARM_DESC(mousepoll, "Polling interval of mice");
 
@@ -1097,7 +1097,9 @@ static int usbhid_start(struct hid_device *hid)
 		/* Change the polling interval of mice and joysticks. */
 		switch (hid->collection->usage) {
 		case HID_GD_MOUSE:
-			if (hid_mousepoll_interval > 0)
+			if (hid_mousepoll_interval == ~0 && interval < 16)
+				interval = 16;
+			else if (hid_mousepoll_interval != ~0 && hid_mousepoll_interval != 0)
 				interval = hid_mousepoll_interval;
 			break;
 		case HID_GD_JOYSTICK:
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1146_325f5f0ec83190f5da79a46600247e8dafc2fae6.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1146_325f5f0ec83190f5da79a46600247e8dafc2fae6.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
